R/score.R

```diff
@@ -177,6 +177,29 @@ score.sgdnet_mgaussian <- function(fit,
          mae = colMeans(apply(abs(y_hat - y), 3, colSums)))
 }
 
+#' @rdname score
+#' @export
+score.sgdnet_poisson <- function(fit,
+                                 x,
+                                 y,
+                                 type.measure = c("deviance", "mse", "mae"),
+                                 s = fit$lambda,
+                                 ...) {
+  type.measure <- match.arg(type.measure)
+  y <- as.vector(y)
+
+  y_hat <- stats::predict(fit, x, s = s)
```

Comment:
Maybe import the generic from `stats` instead? I imagine we're using it pretty frequently already.

Reply:
Hi mentor, may I ask how to do this?

Reply:
Just add `roxygen2` comment `@importFrom stats predict` and update the `NAMESPACE` file. It should "just work" after that.